File: solution/1400-1499/1460.Make Two Arrays Equal by Reversing Sub-arrays/Solution.ts
function canBeEqual(target: number[], arr: number[]): boolean {
    const n = target.length;
    const count = new Array(1001).fill(0);
    for (let i = 0; i < n; i++) {
        count[target[i]]++;
        count[arr[i]]--;
    }
    return count.every(v => v === 0);
}

File: solution/1400-1499/1460.Make Two Arrays Equal by Reversing Sub-arrays/Solution.c
bool canBeEqual(int* target, int targetSize, int* arr, int arrSize){
    int count[1001] = {0};
    for (int i = 0 ; i < targetSize; i++) {
        count[target[i]]++;
        count[arr[i]]--;
    }
    for (int i = 0; i < 1001; i++) {
        if (count[i] != 0) {
            return false;
        }
    }
    return true;
}
